src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -319,7 +319,7 @@ const PublicationReleasePage: NextPage<Props> = ({ data }) => {
               );
             }}
           >
-            <AccordionSection heading="Download data">
+            <AccordionSection heading="Download data and files">
```

Comment:
This is going to break a bunch of UI tests. Make sure to fix these once this has been merged back into dev.